ftest-maven/quarkus-example/pom.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.acme</groupId>
+  <artifactId>quarkus-example</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <properties>
+    <surefire-plugin.version>2.22.0</surefire-plugin.version>
+    <quarkus.version>0.14.0</quarkus.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-bom</artifactId>
+        <version>${quarkus.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-hibernate-orm-panache</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-postgresql</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-junit5</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-maven-plugin</artifactId>
+        <version>${quarkus.version}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>build</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${surefire-plugin.version}</version>
+        <configuration>
+          <systemProperties>
+            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+          </systemProperties>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>io.jshift.odo</groupId>
+        <artifactId>odo-maven-plugin</artifactId>
+        <version>1.0.0-beta2-SNAPSHOT</version>
+        <configuration>
+          <createComponent>
+            <maxMemory>2</maxMemory>
+          </createComponent>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>io.jshift.odo</groupId>
+            <artifactId>odo-binary</artifactId>
+            <version>1.0.0-beta2-SNAPSHOT</version>
```

Comment:
It would be better if `${project.version}` is used

Reply:
We cannot do this because in this context the project.version is the version of this project and not the odo-java project.

Reply:
umm, then how does it work in `simple/` project? It has got version set to project version

Reply:
because the first time both odo-java and thiss imple had the same version set

Reply:
when we automate the run of these tests we would need to substitute them. I think that FMP is doing something as a regexp replacement.